applications/FluidDynamicsApplication/python_scripts/navier_stokes_two_fluid_hydraulic_solver.py

```diff
@@ -653,7 +658,13 @@ def _CreateScheme(self):
             domain_size + 1)
 
         return scheme
-
+    
+    def _HydraulicBoundaryConditionCheck(self,boundary,name):
+        # Check if the inlet and outl 
+        computing_model_part = self.GetComputingModelPart()
+        not_boundary_nodes=any([node.Is(boundary) for node in computing_model_part.Nodes])
```

Comment:
```suggestion
        not_boundary_nodes=not any([node.Is(boundary) for node in computing_model_part.Nodes])
```

I think it should be like this, but please check the logic yourself, I only wanted to point out a potentially smarter way